tutorials/ksample.py

```diff
@@ -164,6 +164,77 @@
 # Any addition, if the bias variant of the test statistic is required, then the ``bias``
 # parameter can be set to ``True``. In general, we do not recommend doing this.
 # Otherwise, these tests runs like :ref:`any other test<general indep>`.
+########################################################################################
+# Smooth Characteristic Function Test
+#
+# The Smooth Characteristic Function Test, or Smooth CF, is a form of non-parametric two-sample
```

Comment:
link to the back to reference documentation using `:class:` like used in the above tutorials

Reply:
links added